event_lead_event/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+
+================
+Event Lead Event
+================
+
+This module extends the functionality of `event` and `website_event_track`.
```

Comment:
ok, seems like you love this sentence each time :smile: 
Keep it but then rephrase the 2nd one:
```
This module extends the functionality of `event` and `website_event_track`
to allow setting and event as "lead event" and promote it.
You can use the flag `lead_event` to put it on top of search results or on the homepage.